src/main/java/frc/robot/DriverJoystick.java

```diff
@@ -88,6 +90,9 @@ private DriverJoystick() {
         super.aButton
                 .onTrue(new InstantCommand(vision::enable0, vision))
                 .onFalse(new InstantCommand(vision::disable0, vision));
+
+        leftTrigger.onTrue(new LiftExtendCommand());
+        rightTrigger.onTrue(new LiftDepressCommand());
```

Comment:
Shouldn't these be wile true? that way if they let go of the button the command will end.
With onTrue, the command won't end until isFinished returns true